SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -128,6 +130,31 @@ public IEnumerable<string> GetInstalledPlugins()
             return plugins.Select(plugin => plugin["key"].ToString());
         }
 
+        /// <summary>
+        /// Attempts to download the quality profile in the specified format
+        /// </summary>
+        public bool TryGetProfileExport(string qualityProfile, string language, string format, out string content)
```

Comment:
Consider adding a parameter comment for "format" as it is not clear what this string is / where to get it from